src/app/core/services/github.service.ts

```diff
@@ -379,7 +379,7 @@ export class GithubService {
       map((response) => {
         return response['data'];
       }),
-      catchError((err) => throwError('Failed to fetch assignable users for repo.'))
+      catchError((err) => throwError('Failed to fetch assignable users for repo (getUsersAssignable).'))
```

Comment:
Let's avoid exposing the method name in the error message we show to users, same for the rest of the error messages